opm/autodiff/MissingFeatures.cpp

```diff
@@ -55,6 +89,14 @@ namespace MissingFeatures {
             "VAPPARS", "VISCREF", "WATVISCT",
             "WPAVE", "WPIMULT", "WPITAB", "WTEMP",
             "WTEST", "WTRACER", "ZIPPY2" };
+        std::multimap<std::string, PartiallySupported<std::string> > string_options;
```

Comment:
> I would guess you could skip the last type in angular braces, since that can be inferred.

Not in the constructor unless you give it something to infer *from*.